### ISSUE TITLE
NVDA crashes after a succesfulll install

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#12421]
### Steps to reproduce:
1. Download launcher of the latest Alpha
2. Close currently running copy of NVDA if any.
3. Start NVDA from the previously downloaded launcher, accept license and choose "install" from the launcher dialog.
4. Accept UAC and wait for NVDA to install successfully.
### Actual behavior:
At the end of installation NVDA crashes and the newly installed copy  starts correctly.
[crash_2021.1.zip](https://github.com/nvaccess/nvda/files/6489839/crash_2021.1.zip)
The attached archive contains log at the debug level and crash dump.

### Expected behavior:
No crash.
#### NVDA installed/portable/running from source:
Running from the launcher.
#### NVDA version:
NVDA version alpha-22774,44fabd74

#### Windows version:
Windows 8
#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.4 does not crash
#### If add-ons are disabled, is your problem still occurring?
Tested with a clean config so no add-ons involved.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes - while I haven't run the tool explicitly it is being run implicitly during install.

cc @michaelDCurran  @seanbudd

**Comment by OzancanKaratas** [comment]
Does this problem also occur with silent installation? Please try “--install-silent” command.

**Comment by seanbudd** [comment]
it seems watchdog is not terminating

```python
DEBUG - core._terminate (20:06:53.874) - MainThread (4520):
Terminating watchdog
```

```python
NVDA exit
CRITICAL - watchdog._crashHandler (20:06:55.077) - MainThread (4520):
NVDA crashed! Minidump written to C:\Users\Admin\AppData\Local\Temp\nvda.log\..\nvda_crash.dmp
INFO - watchdog._crashHandler (20:06:57.202) - MainThread (4520):
Listing stacks for Python threads:
Python stack for thread 4520 (MainThread):
  File "watchdog.pyc", line 213, in _crashHandler
  File "watchdog.pyc", line 63, in getFormattedStacksForAllThreads

INFO - watchdog._crashHandler (20:06:57.202) - MainThread (4520):
Restarting due to crash
```

**Comment by seanbudd** [comment]
closed by #12431
